lighthub/inputs.cpp

```diff
@@ -81,31 +81,42 @@ void Input::Parse()
 }
 
 int Input::Poll()
-{ 
+{
   boolean v;
   if (!isValid()) return -1;
 
-  
-  if (inType & IN_ACTIVE_HIGH) 
-      {   pinMode(pin, INPUT);    
-        v = (digitalRead(pin)==HIGH); 
-      } 
-        else 
-      {   pinMode(pin, INPUT_PULLUP);    
+
+  if (inType == 1)
```

Comment:
what reason to move from (inType & IN_ACTIVE_HIGH) ?

Reply:
There is 3 options in my branch: 0 - standard button, 1 - should be trigger according to docs but real behaviour is the same as 0 but with INPUT_PULLUP, 2- is trigger behaviour. I've introduced a new mode for backward compatibility. I did not want to touch 1 mode which can be used in configuration.
IN_ACTIVE_HIGH is 128 so I can't use it to compare with 2.
I'll check develop branch and see how it works without my changes and then I'll fix this pull request.